packages/ui-components/src/components/tag/tag.scss

```diff
@@ -0,0 +1,31 @@
+@import '../../assets/styles/globals';
+
+:host {
+	/**
+	 * @prop --tag-background-color: Tag Background Color.
+	 * @prop --tag-text-color: Tag text Color.
+	 */
+
+	--tag-background-color: #{kv-color('neutral-6')};
+	--tag-text-color: #{kv-color('neutral-2')};
+}
+
+.tag-container {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	width: fit-content;
+	border-radius: 4px;
+	padding: $spacing;
+	background-color: var(--tag-background-color);
+
+	.tag-text {
+		@include kv-font-label-small-uppercase-regular;
+
+		color: var(--tag-text-color);
+
+		&.has-left-slot {
```

Comment:
You don't need this, just add `gap: $spacing`. It will automatically add a gap between all children.

This way you can remove this has left slot logic from the component.